code/game/dna/genes/monkey.dm

```diff
@@ -37,8 +37,8 @@
 
 
 	var/mob/living/carbon/human/monkey/O = null
-	if(H.species.primitive)
-		O = new H.species.primitive(src)
+	if(H.species.primitive_form)

```

Comment:
I'm fairly sure this If won't do what you want it to; primitive IIRC being a var that stores if a mob currently is in primitive_form, and primitive_form storing what that form would be - so that If would fire for every single race who has any primitive_form set to anything not null. Especially considering the primitive var hasn't been colon'd so it has to exist on compiletime.

Then again, I'm going by memory, I didn't look into what the mob vars are in a while.